### PR TITLE
Add message attachment actions

### PR DESCRIPTION
**Pull request by denuno**

### CODE REVIEW

File: plugin/src/main/groovy/net/madeng/gradle/plugin/slack/api/Action.groovy [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    closure.resolveStrategy = Closure.DELEGATE_FIRST
+    confirm = new Confirm()
+    closure.delegate = confirm
+    closure()
+  }
+
+  class Confirm {
+    String title;
+    String text;
+    String ok_text;
```
Comment (jongyoul) [inline plugin/src/main/groovy/net/madeng/gradle/plugin/slack/api/Action.groovy]: I really appreciate your contribution. Your PR looks good to me but I have only a small request for keeping fields' format. Can you change `ok_test` to `okTest` and `dismiss_text` to `dismissText`?

**Comment by jongyoul** [comment]
@denuno Hi, Thanks for contributing my plugin. I'll add some documents for adding your feature after merging your PR. Can you check my comment?

**Comment by jongyoul** [comment]
We can find that example on https://api.slack.com/docs/messages/builder?msg=%7B%22text%22%3A%22Would%20you%20like%20to%20play%20a%20game%3F%22%2C%22attachments%22%3A%5B%7B%22text%22%3A%22Choose%20a%20game%20to%20play%22%2C%22fallback%22%3A%22You%20are%20unable%20to%20choose%20a%20game%22%2C%22callback_id%22%3A%22wopr_game%22%2C%22color%22%3A%22%233AA3E3%22%2C%22attachment_type%22%3A%22default%22%2C%22actions%22%3A%5B%7B%22name%22%3A%22game%22%2C%22text%22%3A%22Chess%22%2C%22type%22%3A%22button%22%2C%22value%22%3A%22chess%22%7D%2C%7B%22name%22%3A%22game%22%2C%22text%22%3A%22Falken%27s%20Maze%22%2C%22type%22%3A%22button%22%2C%22value%22%3A%22maze%22%7D%2C%7B%22name%22%3A%22game%22%2C%22text%22%3A%22Thermonuclear%20War%22%2C%22style%22%3A%22danger%22%2C%22type%22%3A%22button%22%2C%22value%22%3A%22war%22%2C%22confirm%22%3A%7B%22title%22%3A%22Are%20you%20sure%3F%22%2C%22text%22%3A%22Wouldn%27t%20you%20prefer%20a%20good%20game%20of%20chess%3F%22%2C%22ok_text%22%3A%22Yes%22%2C%22dismiss_text%22%3A%22No%22%7D%7D%5D%7D%5D%7D

**Comment by denuno** [comment]
Updated!

**Comment by jongyoul** [comment]
Great! Thanks!

**Comment by jongyoul** [comment]
@denuno Merged and I've published this new feature with 0.0.20. Thanks again.

**Comment by denuno** [comment]
@jongyoul thank you for getting this in so fast, and for the plugin, it fits my needs perfect!

**Pull request merged**